lib/main/localizations/app_localizations.dart

```diff
@@ -4273,4 +4273,32 @@ class AppLocalizations {
       name: 'createTwakeIdFailed',
     );
   }
+
+  String get swithAccountConfirmation {
+    return Intl.message(
+      'Switch Account Confirmation',
+      name: 'swithAccountConfirmation',
+    );
+  }
+
+  String get yesLogout {
+    return Intl.message(
+      'Yes, Log out',
+      name: 'yesLogout',
+    );
+  }
+
+  String get doYouWantToLogoutOf {
```

Comment:
Should change to `You are currently logged in with [current_account_email]. Do you want to log out and switch to [new_email]?`.

Button change to: `Yes, Switch Account`